jax/_src/interpreters/mlir.py

```diff
@@ -429,7 +429,8 @@ class LoweringParameters:
   # The current lowering platforms, a non-empty tuple containing some of
   # 'cpu', 'cuda', 'rocm', 'tpu'. If the tuple has multiple entries we are
```

Comment:
JFYI you could in theory restrict the supported platforms via literal types

```
Platform = Literal['cpu', 'cuda', 'rocm', 'tpu']
```

Reply:
Thanks for the pointer, I have started to do this but it requires quite a few changes in other files that otherwise would not be included here. I should try to do this separately.